package.json

```diff
@@ -1985,6 +1985,29 @@
 						"description": "Whether to show annotations against constructor, method invocations and lists that span multiple lines.",
 						"scope": "window"
 					},
+					"dart.closingLabelsPrefix": {
+						"type": [
+							"null",
+							"string"
+						],
+						"default": null,
+						"markdownDescription": "The prefix to use for closing labels. If `null`, the default ` // ` prefix will be used.",
+						"scope": "window"
+					},
+					"dart.closingLabelsStyle": {
+						"type": "string",
+						"enum": [
+							"normal",
+							"italic"
+						],
+						"enumItemLabels": [
+							"Normal",
+							"Italic"
+						],
+						"default": "normal",
+						"description": "The style to use for closing labels. This setting requires a theme that supports italic text.",
```

Comment:
Same here, "The text style"

> This setting requires a theme that supports italic text.

nit: "Using _Italic_ request a theme that supports it" (although I didn't realise the theme would matter here?)

Reply:
> although I didn't realise the theme would matter here?

I would think so, or at least the font would matter. I can probably say _font_ there as well.

But if you don't believe this is true I could remove this. I could not find a font that didn't had this option to test, if you can, please let me know.